.github/workflows/issue-view.yml

```diff
@@ -0,0 +1,384 @@
+# This is a basic workflow to help you get started with Actions
+
+name: issue-view-workflow
+
+on: [workflow_dispatch]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: windows-latest
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@v2
+
+    - name: 'Generate Dashboard'
+      id: dashboard
+      uses: ethomson/issue-dashboard@v1
+      with:
+        config: |
+          title: 'Issues detailed view'
+          output:
+            format: html
+            filename: 'issue.html'
+          setup: |
+              userdata.count = async function(repo,label) {
+                function sleep(milliseconds) {
+                  const date = Date.now();
+                  let currentDate = null;
+                  do {
+                    currentDate = Date.now();
+                  } while (currentDate - date < milliseconds);
+                }
+                sleep(2000)
```

Comment:
why is sleep required? how do we ensure this sleep time will suffice always? Can we avoid sleep?

Reply:
Due to increase in number of queries per minute  by adding more actions, we are adding sleep to avoid **_API rate limit exceeded error_**  This ensures 2s sleep time for every query.

Reply:
Just adding on to it, there is a limit on the number of calls that can be made to the GitHub APIs in a minute.
As here, we'll be using multiple repos with different label filters, sleep ensures we don't run into error. :)

Reply:
ok, what is the limit number exactly? And just to confirm this issue we are facing only for details and not in summary page, right?